_dashboards/get-started/index.md

```diff
@@ -11,15 +11,15 @@ redirect_from:
 
 {%- comment -%}The `/docs/opensearch-dashboards/` redirect is specifically to support the UI links in OpenSearch Dashboards 1.0.0.{%- endcomment -%}
 
-# Getting started with OpenSearch Dashboards
+# Get started with OpenSearch Dashboards
 
 OpenSearch Dashboards is an open-source, integrated visualization tool that makes it easy for users to explore their data in OpenSearch. From real-time application monitoring, threat detection, and incident management to personalized search, OpenSearch Dashboards gives you the data visualizations needed to graphically represent trends, outliers, and patterns in your data. The image below shows a sample of data visualizations in OpenSearch Dashboards.  
 
 <img src="{{site.url}}{{site.baseurl}}/images/dashboard-flight.png" alt="User interface showing several data visualizations">
```

Comment:
Consider adding `{:img-fluid}` at the end here and to other image links in the document to make them responsive.

Reply:
I'm addressing the navigation renumber after 2.5 to avoid conflicts with merging.